modules/iam-identity/src/test/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentityIT.java

```diff
@@ -143,7 +89,19 @@ public class IamIdentityIT extends SdkIntegrationTestBase {
 
     private String reportReference;
     private String reportReferenceMfa;
-
+    
+    private String profileTemplateId;
```

Comment:
These "id" and "etag" variables could be automatically managed in the generated code for you if you add links to your API definition that describe the inter-operation dependencies (e.g. create_api_key supplies the "id" value from its response to the other api_key-related operations that need an api_key id.  Please contact me for details if interested.